easybuild/easyconfigs/p/Python/Python-2.7.8-intel-2015a.eb

```diff
@@ -0,0 +1,94 @@
+name = 'Python'
+version = '2.7.8'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# libffi build in python is still broken, see http://bugs.python.org/issue4130
+patches = ['python-2.7_libffi-include-xmmintrin.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
```

Comment:
include the commented out OpenSSL dep, together with the comment explaining why the osdeps approach is preferred for OpenSSL?